ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -152,7 +152,7 @@ openstack*)
     source "${SHARED_DIR}/cinder_credentials.sh"
     export TEST_PROVIDER='{"type":"openstack"}';;
 ovirt) export TEST_PROVIDER='{"type":"ovirt"}';;
-equinix-ocp-metal|equinix-ocp-metal-qe)
```

Comment:
We need to add this in the other files within the openshift-extended folder.

Reply:
Added a check for powervs-1 in all files.